workspaces/marketplace/examples/plugins/tekton.yaml

```diff
@@ -14,6 +14,8 @@ spec:
   developer: Red Hat
   lifecycle: production
   owner: redhat
+  packages:
+    - 'backstage-community-plugin-tekton'
```

Comment:
Can you rebase and revert this change. It's already added at the end of the file.